eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -202,8 +215,43 @@ private Collection<Dependency> deps() throws IOException {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Check if all dependencies have transitive dependencies.
+     *
+     * @param deps Dependencies
+     * @throws java.lang.IllegalStateException if a transitive dependency is found
+     */
+    private void checkTransitive(final Collection<Dependency> deps) {
+        if (this.checkTransitive) {
+            deps.forEach(
+                dep -> {
+                    if (!new DcsFiltered(
+                        new DcsDepgraph(
+                            project,
+                            session,
+                            manager,
+                            this.targetDir.toPath()
+                                .resolve(ResolveMojo.DIR)
+                                .resolve("dependencies-info"),
+                            dep
+                        ),
+                        Arrays.asList(
+                            new DcsFiltered.NotRuntime(),
+                            new DcsFiltered.NotSame(dep),
+                            new DcsFiltered.NotTesting()
+                        )
+                    ).all().isEmpty()) {
+                        throw new IllegalStateException(
+                            String.format("%s contains transitive dependencies", dep)
+                        );
+                    }
+                });
```

Comment:
@volodya-lombrozo parenthesis should be on new line here

Reply:
@mximp fixed